### PR TITLE
Get kindergarten comments

### PR DESCRIPTION
**Pull request by Israa91**
# Get Comments by kindergarten ID
Relates #13

## Database Query
- In file Rawdaty/server/database/queries/comments/getCommentsQuery.js
  - create database query which returns the comments data according to the kindergarten id , the returened data are :
        -  comment_id
        -  kindergarten_id
        - user_name,
        - comment,
        -  rating,
        -  create_at
         
## Controller
 - Add getComments which using the above query to return the required data.

## Tests
- Add one database query test
- Add four route tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 .env
+coverage
```
Comment (Mu7ammadAbed) [inline .gitignore:3]: new line

File: server/router/index.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 const router = require('express').Router();
 const kindergarten = require('./kindergarten');
+const comment = require('./comment');
```
Comment (Mu7ammadAbed) [inline server/router/index.js:3]:
```suggestion
const comments = require('./comments');
```

File: test/comment.test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      .then((result) => expect(result.rows).toHaveLength(4))
+      .catch());
+});
+
+// test the route /kindergarten/:kindergartenId/comments
+describe('Test the route /kindergarten/:kindergartenId/comments', () => {
+  test('should return status code 200 and data length 2 when given GET  /kindergarten/2', async () => {
+    expect.assertions(1);
+    const res = await request(app)
+      .get('/api/v1//kindergarten/2/comments')
```
Comment (Mu7ammadAbed) [inline test/comment.test.js:22]:
```suggestion
      .get('/api/v1/kindergarten/2/comments')
```

File: test/comment.test.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      .expect(200);
+    expect(res.body.data.length).toBe(2);
```
Comment (Mu7ammadAbed) [inline test/comment.test.js:24]:
```suggestion
    const { data } = res.body;
    expect(data.length).toBe(2);
```

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

**Review by Mu7ammadAbed — APPROVED** [review]

**Review by Osama-you — APPROVED** [review]

**Review by Alaalser — APPROVED** [review]

**Review by alaa-sahloub — APPROVED** [review]

**Pull request merged**